include/glow/Importer/ONNX.h

```diff
@@ -48,28 +48,47 @@ class ONNXModelLoader
   /// Load the network initializers from the GraphProto.
   void loadInitializers(onnx::GraphProto &net);
 
+  /// \returns true if operator \p op can be loaded.
   /// Load the operator \p op into the network. This creates one or more nodes
   /// in the network.
-  void loadOperator(const onnx::NodeProto &op);
+  bool loadOperator(const onnx::NodeProto &op);
 
-  /// Reads a network (weights or structure) from the serialized protocol buffer
-  /// file.
-  bool loadProtoFile(onnx::GraphProto &net, const std::string &filename);
+  /// \returns true if serialized protobuf can be read from the \p filename.
+  /// Loads GraphProto \p net containing network (weights or structure)
```

Comment:
I am not sure I understand the meaning here.
Maybe `loads into \p net the network...`